tests/presentation/verify.no-auth.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { test } from '@playwright/test';
```

Comment:
Tests missing here, this is definitely in scope of something we can check right now.